lib/rules/label-content-name-mismatch.json

```diff
@@ -0,0 +1,17 @@
+{
+  "id": "label-content-name-mismatch",
+  "matches": "label-content-name-mismatch-matches.js",
+  "tags": [
+    "wcag21a",
+    "wcag253"
```

Comment:
We should make this rule experimental for the time being. I don't trust that this isn't going to give false positives.